.github/workflows/platform.publish-module-index-json.yml

```
@@ -49,11 +55,26 @@ jobs:
             ErrorAction                                  = 'Continue'
           }
 
-          Write-Verbose "Invoke task with" -Verbose
-          Write-Verbose ($functionInput | ConvertTo-Json | Out-String) -Verbose
+          if ('${{ (fromJson(inputs.doNotMergeWithLastModuleIndexJsonFileVersion)) }}' -eq '$true' ) {
+            Write-Verbose "Invoke task with" -Verbose
+            Write-Verbose ($functionInput | ConvertTo-Json | Out-String) -Verbose
+            Write-Verbose "doNotMergeWithLastModuleIndexJsonFileVersion value is..." -Verbose
+            Write-Verbose ('${{ (fromJson(inputs.doNotMergeWithLastModuleIndexJsonFileVersion)) }}' | ConvertTo-Json | Out-String) -Verbose
 
-          if(-not (Invoke-AvmJsonModuleIndexGeneration @functionInput)) {
-            Write-Output ('{0}={1}' -f 'anyErrorsOccurred', $true) >> $env:GITHUB_ENV
+            if(-not (Invoke-AvmJsonModuleIndexGeneration -doNotMergeWithLastModuleIndexJsonFileVersion @functionInput)) {
+              Write-Output ('{0}={1}' -f 'anyErrorsOccurred', $true) >> $env:GITHUB_ENV
+            }
+          }
+
+          if ('${{ (fromJson(inputs.doNotMergeWithLastModuleIndexJsonFileVersion)) }}' -eq '$false' ) {
```

```suggestion
          } else {
```

Can't really be anything else, I guess